libs/server/Resp/Bitmap/BitmapCommands.cs

```diff
@@ -617,14 +617,15 @@ private bool StringBitField<TGarnetApi>(int count, byte* ptr, ref TGarnetApi sto
 
             for (int i = 0; i < secondaryCmdCount; i++)
             {
+                /* Commenting due to excessive verbosity
```

Comment:
May be these could be changed to more verbose log levels like Trace.

Reply:
even Trace is too much for these, seems like. we cannot log every interaction.